lib/internal/errors.js

```diff
@@ -53,9 +52,20 @@ class AssertionError extends Error {
   }
 }
 
+// This is defined here instead of using the assert module to avoid a
+// circular dependency. The effect is largely the same.
+function assert(condition, message) {
+  if (!condition) {
+    throw new AssertionError({
+      message,
+      actual: false,
+      expected: true,
+      operator: '=='
```

Comment:
This should be `===`.

Reply:
IMHO since this is an internal module assertions should be made in `/test/`, so there's no need to runtime assertions at all.